.gitlab-ci.yml

```diff
@@ -446,11 +455,11 @@ doc:refman:deploy:
       - $DOCUMENTATION_DEPLOY_KEY
   dependencies:
     - doc:ml-api:odoc
-    - doc:refman:dune
+    - doc:refman-html:dune
```

Comment:
IMHO not worth doing this renaming.

Reply:
This is more consistent with the name of the targets though.

Reply:
Not worth the conflict with future PRs IMO, the `dune` naming is gone after #8729 for example.

Reply:
I see, thanks.